test/intl402/NumberFormat/default-currency-maximum-fraction-digits.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 the V8 project authors. All rights reserved.
```

Comment:
nit: this file can be renamed to `dft-currency-mnfd-range-check-mxfd.js`.

It should not block this PR as well.